pkg/api/api.proto

```diff
@@ -206,6 +211,20 @@ message StopContainerResponse {
   repeated ContainerUpdate update = 1;
 }
 
+message UpdatePodSandboxRequest {
+  // Pod being updated.
+  PodSandbox pod = 1;
+  // Overhead associated with this pod.
+  LinuxResources overhead_linux_resources = 2;
+  // Sum of container resources for this pod.
+  LinuxResources linux_resources = 3;
+}
+
+message UpdatePodSandboxResponse {
+  // Requested update to pod.
+  PodSandboxUpdate update = 1;
```

Comment:
Considering that pod level cgroup handling is in the kubelet I think this is a mere informative event and the message should reflect it. There is nothing the runtime could do if a plugin decided to request changes to the pod resource update. Then the response message should reflect this, and be simply empty.

Reply:
Would you prefer this uses `Empty{}`, or an empty `UpdatePodSandboxResponse`? I figure the latter is more extensible in case pod-level cgroup handling moves more to the container runtime in the future.

Reply:
Yes and that's why I'd prefer to use a dedicated `UpdatePodSandboxResponse`, if other reviewer will agree with that. It might be a bit inconsistent compared to the rest of the pod handling RPC calls and messages for the time being. But I'd like to anyway get rid of `Empty` (and the usage of `StateChange` on the wire) altogether before we tag an NRI 1.0.0 version. It's just better to hold that change back until all those pending PRs are merged which also touch the wire protocol, to avoid unnecessary conflicts.